src/main/java/uk/gov/digital/ho/hocs/domain/exceptions/ApplicationExceptions.java

```diff
@@ -23,4 +23,125 @@ public LogEvent getEvent() {
         public LogEvent getException() {return exception;}
 
     }
+
+    class DocumentCreationException extends RuntimeException {
```

Comment:
This is just following what was already here - probably need to create a standard for these that works.